cityofaustin/atd-moped
# Add columns to project view

## Associated issues

This PR adds task order, contractor and project DO# to the project list. You can also filter projects by those fields. 


## Testing
**URL to test:** 

https://moped-test.austinmobility.io/moped/ 

There are database changes, so please use moped test or run locally. 

**Steps to test:**

<img width="1193" alt="Screen Shot 2022-04-15 at 4 43 28 PM" src="https://user-images.githubusercontent.com/12474808/163639989-e275fb2f-7d33-4696-b1f9-010b198f6b46.png">

Toggle columns to show or hide. Using advanced search for the columns works, however whenever the page reloads, the default column view is shown. 

---
#### Ship list
